fe/fe-core/src/main/java/com/starrocks/metric/MaterializedViewMetricsRegistry.java

```diff
@@ -71,40 +69,34 @@ public static void collectMaterializedViewMetrics(MetricVisitor visitor, boolean
             if (null == db) {
                 continue;
             }
-            Locker locker = new Locker();
-            locker.lockDatabase(db, LockType.READ);
-            try {
-                for (MaterializedView mv : db.getMaterializedViews()) {
-                    MaterializedViewMetricsEntity mvEntity =
-                            MaterializedViewMetricsRegistry.getInstance().getMetricsEntity(mv.getMvId());
+            for (MaterializedView mv : db.getMaterializedViews()) {
```

Comment:
add `collectMaterializedViewMetrics` to `MetricCalculator`, 
otherwise, when the hash table is resized, get_metric will be stuck for a long time, thus falsely reporting FE is
 down?